cli/cache.rs

```diff
@@ -14,18 +14,22 @@ use deno_graph::source::LoadFuture;
 use deno_graph::source::LoadResponse;
 use deno_graph::source::Loader;
 use deno_runtime::permissions::Permissions;
+use std::collections::HashSet;
 use std::sync::Arc;
 
 #[derive(Debug, Deserialize, Serialize)]
 pub struct EmitMetadata {
   pub version_hash: String,
+  pub type_checks_succeeded: HashSet<String>,
```

Comment:
This name feels like a boolean. Any other ideas? Perhaps `type_check_hashes` with a comment saying it only contains successful hashes?